e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/AwaitHelper.java

```diff
@@ -10,7 +10,7 @@
 
 public class AwaitHelper {
 
-    protected static final int WAIT_FOR_IN_SECONDS = 60;
+    protected static final int WAIT_FOR_IN_SECONDS = 90;
```

Comment:
I think this is for slower machines? ;) At some point we would need to stop raising that value. for now ok.

Reply:
I've reduced this again, will review it during the next pipeline run